tasks/stacks/build-stack-image-windows/run

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+build_args=$(cat "${BUILD_ARGS_FILE}" | awk '{printf " --build-arg %s", $0}')
+echo "Build args:"
+echo ${build_args}
+docker build${build_args} --no-cache -t img "${CONTEXT}"
```

Comment:
Minor detail but does not having a space between `build` and `${build_args}` cause an error?

Reply:
@TisVictress Good eye! It's actually intentional, as the `build_args` string itself starts with a space